code/__DEFINES/flags.dm

```diff
@@ -24,6 +24,9 @@
 
 #define DROPDEL					16384	// When dropped, it calls qdel on itself
 
+///Whether or not this atom shows screentips when hovered over
+#define NO_SCREENTIPS_1			32768
```

Comment:
```suggestion
#define NO_SCREENTIPS			32768
```
We dont use `flags_1` and `flags_2`.

Reply:
ah, i'd caught the _2 things but yeah, I can tidy that up